### PR TITLE
[W4.5c][W13-A1]Raynold Ng

### PR DESCRIPTION
**Pull request by raynoldng**
LO-Exceptions

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -111,6 +111,10 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (StorageOperationException e) {
```
Comment (okkhoy) [inline src/seedu/addressbook/Main.java:114]: In general, it is good to handle the exception closer to where it occurs.

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -111,6 +111,10 @@ private CommandResult executeCommand(Command command)  {
...
+            ui.showToUser(e.getMessage());
+            ui.showToUser("Storage file is read only, changes may not be permanent");
```
Comment (okkhoy) [inline src/seedu/addressbook/Main.java:116]: This is one of the ways of handling the exception.
Think how you can improve the user experience in light of the exception that just occurred, than just showing an error message.

**Comment by okkhoy** [comment]
@raynoldng 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/Main.java (hunk 2 above)
Comment (raynoldng) [reply src/seedu/addressbook/Main.java:116]: I suppose the app can continue running instead of terminating, but only view features will be available?

**Comment by raynoldng** [comment]
Thank you for the comments. WIll bear it in mind for future LOs

**Pull request closed without merging**